roles/manage_agents/tasks/add_agents.yml

```
@@ -16,7 +16,7 @@
             name: "agent-{{ manage_agents_resource_class }}-lock"
             namespace: "{{ default_agent_namespace }}"
             labels:
-              cloudkit.openshift.io/aap-job-id: "{{ awx_job_id|default('unknown') }}"
+              cloudkit.openshift.io/aap-job-id: "job-{{ awx_job_id | default('unknown') }}"
```

```suggestion
              cloudkit.openshift.io/aap-job-id: "{{ awx_job_id | default('unknown') | str }}"
```

have you tried cast to `str`, I had to do the opposite in the Nodepool to force an `int` instead to a `str`